### ISSUE TITLE
Request to add new papers

### ISSUE THREAD

**Issue by WeiXiongUST** [LightChen233/Awesome-Long-Chain-of-Thought-Reasoning#9]
Hi thanks for the great survey. 

I am creating this issue to bring your attention to our works

1. Raft: Reward ranked finetuning for generative foundation model alignment, 2023.4. We present the iterative rejection sampling fine-tuning in this work, some of the mentioned methods in the survey that claim to be concurrent to us are actually ~5 month later than us.
2. Building Math Agents with Multi-Turn Iterative Preference Learning, 2024.9. We derive the optimality condition of the multi-turn reasoning in the face of external observation/message and derive a multi-turn version of dpo for math agent (with python interpreter) training. 

Many thanks in advance.

**Comment by LightChen233** [comment]
Thank you for your attention to our work. Our latest version has updated your mentioned works :)
